src/responder/kcm/kcmsrv_ops.c

```diff
@@ -161,7 +161,7 @@ struct tevent_req *kcm_cmd_send(TALLOC_CTX *mem_ctx,
                                         state,
```

Comment:
I think that the `to the new allocated iobuffer` should be `newly allocated...`. (in the commit message).

Reply:
Fixed